File: C#/백준/Silver/29934. Important Messages/Important Messages.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int cnt =0;
            String[] aa = new String[a];
            for (int i=0;i<a;i++){
                aa[i] = Console.ReadLine();
            }
            int b = int.Parse(Console.ReadLine());
            for (int i=0;i<b;i++){
                String s = Console.ReadLine();
                for (int j=0;j<a;j++){
                    if(s.Equals(aa[j])){
                        cnt++;
                        break;
                    }
                }
            }
            Console.WriteLine(cnt);
        }
    }
}

File: Java/백준/Silver/29934. Important Messages/Important Messages.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int cnt = 0;
        String[] aa = new String[a];
        for (int i=0;i<a;i++){
            aa[i] = in.next();
        }
        int b = in.nextInt();
        for (int i=0;i<b;i++){
            String s = in.next();
            for (int j=0;j<a;j++){
                if(s.equals(aa[j])){
                    cnt++;
                    break;
                }
            }
        }
        System.out.println(cnt);
    }
}